src/Command/GenerateKeysCommand.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Helis\SettingsManagerBundle\Command;
+
+
+use ParagonIE\Paseto\Protocol\Version2;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class GenerateKeysCommand extends Command
+{
+
+    protected function configure(): void
+    {
+        $this
+            ->setName('debug:settings:generate-keys')
```

Comment:
Based on command class name its command name should be `helis:settings:generate-keys`